File: official/docs/java/users/CreateChildUser.java
import com.easypost.EasyPost;
import java.util.HashMap;

public class CreateChildUser {
    public static void main(String[] args) {
        EasyPost.apiKey = System.getenv("EASYPOST_API_KEY");
        
        HashMap<String, Object> userMap = new HashMap<String, Object>();
        userMap.put("name", "Child Account Name");

        User user = User.create(userMap);

        System.out.println(user);
    }
}

File: official/docs/csharp/users/create-child-user.cs
using System;
using System.Collections.Generic;
using System.Threading.Tasks;
using Newtonsoft.Json;
using EasyPost;

namespace EasyPostExamples
{
    public class Examples
    {
        public static async Task Main()
        {
            string apiKey = Environment.GetEnvironmentVariable("EASYPOST_API_KEY")!;

            EasyPost.ClientManager.SetCurrent(apiKey);

            User user = await User.Create(new Dictionary<string, object>()
            {
                {
                    "name", "Child Account Name"
                }
            });

            Console.WriteLine(JsonConvert.SerializeObject(user, Formatting.Indented));
        }
    }
}
